README.md

```diff
@@ -565,11 +565,22 @@ Please use the [GitHub issues page](https://github.com/jonbgallant/azure-iot-edg
 
 Please fork, branch and pull-request any changes you'd like to make.
 
-Run the following command to install all the dependencies needed to build iotedgedev and run the tests.
+#### Contributor Dev Machine Setup
+1. Install **[Microsoft Visual C++ 14.0](http://landinghub.visualstudio.com/visual-cpp-build-tools)**
 
-`pip install -r requirements_dev.txt`
+1. Install **OpenSSL 1.1.0g**
+    - Windows - [Install from OpenSSL's website](https://www.openssl.org/source/)
+        > after download, Save it in>C:\OpenSSL-Win64\ 
```

Comment:
Please include more instructions on what exactly the user needs to do.  The link doesn't really do a good job of explaining that.

1. Download .tar.gz file
2. Extract to c:\..

etc